README.md

```diff
@@ -82,5 +113,5 @@ GET /files/a123b1234
 
 ##Put a file
 ````
-PUT /files/a123b1234
+PUT /files/a123b1234?length=nn&offset=nn
```

Comment:
How do you recommend using the OPTIONS method to indicate that this header is supported?  That's really all I want to do.  Yeah, I forgot you had suggested the header ;-)